google_shopping.js

```diff
@@ -1,5 +1,66 @@
 var data = require("./products.json")
 
-// Write your solutions below
+var i;
 
-console.log(data["items"]);
\ No newline at end of file
+var items = data['items'];
+// var title = data['items'][0].product.title;
+// var brand = data['items'][0].product.brand;
+//these exist but I can't figure out how to make them accept [i] :(
+
+//#1
+
+// var counter = 0;
+
+// for(var i = 0; i < items.length; i++) {
+// 	if (items[i].kind === 'shopping#product') {
+// 		counter++;
+// 	}
+// }
+
+// console.log(counter);
+
+//Where else is this count information stored in the search results? -> currentItemCount 
+
+
+//#2
+
+// for(var i = 0; i < items.length; i++) {
+//     if (items[i].product.inventories[0].availability === "backorder") 
+//       console.log(items[i].product.title);
+//   }
+
```

Comment:
Missing final closing curly brace for the For loop. Great job on the questions though! Keep it up!